### PR TITLE
कोड गुणवत्ता की मरम्मत

### PR DESCRIPTION
**Pull request by KumarGanesha1996**
## i see you project....
.
.
.
.
.
.
### its very good....!!!!!!_**

regards,
Kumar

### CODE REVIEW

File: src/ui/UtilityFilesReader.java [hunk 1]
```diff
@@ -11,30 +11,27 @@
...
-            while(reader.hasNextLine()){
-                String line = reader.nextLine();
-                String[] tmp = line.split(" as ");
-                String name = tmp[0].trim();
-                if(tmp[0].contains(" ") || tmp[0].contains("\t"))
-                    throw new UtilityFileException("multiple words before \"as\"");
-                String[] aliases = tmp[1].split(",");
-                for(String alias : aliases)
-                    socketAliases.put(alias.trim(), name);
+            try (Scanner reader = new Scanner(socketAliasesFile)) {
```
Comment (KumarGanesha1996) [inline src/ui/UtilityFilesReader.java:22]: ### importrant change..!!!

File: src/ui/UtilityFilesReader.java [hunk 2]
```diff
@@ -11,30 +11,27 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-/**
```
Comment (KumarGanesha1996) [inline src/ui/UtilityFilesReader.java:14]: i improve quality... no loger bad code alert 😂

Comment (KumarGanesha1996) [reply src/ui/UtilityFilesReader.java:14]: **@EndRay please review fast.... i need merge at two days...........................**

**Comment by watislaf** [comment]
nice work

**Pull request closed without merging**